common/src/main/scala/explore/components/TileController.scala

```diff
@@ -72,6 +73,19 @@ object TileController {
       .filter(_.i.forall(_ === id.value))
       .andThen(layoutItemHeight)
 
+  def tileResizable(id: Tile.TileId): Traversal[LayoutsMap, Boolean | Unit] =
```

Comment:
Why `Boolean | Unit`?

Reply:
Yeah that isn't nice, it comes form the facade so I should fix there